db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStorageServices < ActiveRecord::Migration[5.2]
+  def change
+    create_table :storage_services do |t|
+      t.string :name
+      t.string :description
+      t.string :uuid
+      t.bigint :version
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
+
+      t.timestamps
+    end
```

Comment:
Same with this model, STI will be helpful

Reply:
Do we need the type column on physical_storage_families ?

Reply:
Yeah I considered that and I don't think so because this is basically a list of possible types in each record.

If they want to do any `supports?(feature)` on the `physical_storage_family` though then it definitely should.

Reply:
@agrare , we don't intend to have a  list of options in each record. Each record would represent a single family in a single provider. If a provider's families are ['SVC', 'Netapp', 'V7000'], it would have three records. One for each family.

That said I don't think we'll be doing STI with physical_storage_families at this point.

Reply:
Yes exactly @gregoryb-at-ibm each one of these records would be the equivalent to what and STI subclass would give us.

I think we still want a `:type` column on this `storage_services` model though right?

Reply:
Oops, must have missed it.